code/_onclick/ai.dm

```diff
@@ -121,6 +121,26 @@
 	target.AICtrlClick(src)
 
 
+/// Reimplementation of base_click_alt for AI
+/mob/living/silicon/ai/proc/ai_base_click_alt(atom/target)
+	// If for some reason we can't alt click
+	if(SEND_SIGNAL(src, COMSIG_MOB_ALTCLICKON, target) & COMSIG_MOB_CANCEL_CLICKON)
+		return
+
+	if(!isturf(target) && can_perform_action(target, (target.interaction_flags_click | SILENT_ADJACENCY)))
+		// Signal intercept
+		if(SEND_SIGNAL(target, COMSIG_CLICK_ALT, src) & CLICK_ACTION_ANY)
+			return
+
+		// AI alt click interaction succeeds
+		if(target.ai_click_alt(src) & CLICK_ACTION_ANY)
+			return
+
+	var/turf/tile = get_turf(target)
+
+	client.loot_panel.open(tile)
```

Comment:
Have you tested this with AI static? 

Like can you open the loot panel over tiles you don't have camera view of?

Reply:
yes i have